src/matbench_genmetrics/utils/match.py

```diff
@@ -78,7 +86,7 @@ def cdvae_cov_struct_fingerprints(structures, verbose=False):
     my_tqdm = get_tqdm(verbose)
     struct_fps = []
     # base_10_check = [10 ** j for j in range(0, 20)]
-    for i, s in enumerate(my_tqdm(structures)):
+    for s in my_tqdm(structures):
         # if i in base_10_check == 0:
         #     logger.info(f"{time()} Struct fingerprint {i}/{len(structures)}")
         site_fps = [CrystalNNFP.featurize(s, i) for i in range(len(s))]
```

Comment:
you might want to consider `featurize_many` or just use `concurrent.futures` and parallelize yourself.
Also, there is a `SiteStatsFingerprint` https://hackingmaterials.lbl.gov/matminer/matminer.featurizers.structure.html#matminer.featurizers.structure.sites.SiteStatsFingerprint

Reply:
Went with `featurize_dataframe`. Thanks for pointing these out! VS Code+debugging+multiprocessing doesn't play nicely together unless the call to parallel code is nested inside a `if __name__ == "__main__":` statement, so you might see those crop up in a few places.